ccd-definition/AuthorisationComplexType.json

```diff
@@ -190,5 +190,29 @@
     "ListElementCode": "judgeFullName",
     "UserRole":"[BARRISTER]",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "respondents1",
+    "ListElementCode": "representedBy",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "respondents1",
+    "ListElementCode": "leadRespondentIndicator",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "respondents1",
+    "ListElementCode": "party",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
```

Comment:
@Jamesnelson117 is it the duplication you mentioned? Could we fix this as separate PR, I wouldn't expect those changes to be here!

Reply:
Yes exactly this and yes agreed i'll make separate PR.